src/components/shared/Button/PaneToggle.css

```diff
@@ -1,18 +1,21 @@
 .toggle-button-start,
 .toggle-button-end {
-  position: absolute;
-  width: 16px;
-  height: 16px;
   transition: transform 0.25s ease-in-out;
-  margin: 0 4px;
   cursor: pointer;
+  align-self: flex-end;
+  padding: 4px 2px;
 }
 
 .toggle-button-start svg,
 .toggle-button-end svg {
+  width: 14px;
   fill: var(--theme-comment);
 }
 
+.toggle-button-end {
+  margin-left: auto;
```

Comment:
This is a hack to right align the `toggle-button-end`. @Garbee any other way to right align only one flex child?